config/config_test.go

```diff
@@ -730,7 +727,6 @@ type MockupOuputPlugin struct {
 
 func (m *MockupOuputPlugin) Connect() error                        { return nil }
 func (m *MockupOuputPlugin) Close() error                          { return nil }
-func (m *MockupOuputPlugin) Description() string                   { return "Mockup test output plugin" }
```

Comment:
Was there a discussion and/or need to remove `Description` at the same time?

Reply:
Why I think it should be removed is that `Description` is only being used when generating the sample config, adding it as a comment above the config. Originally I was looking into keeping the function and using `{{ .Description }}`, moving the text from the plugin to a separate section in the README.md to extract it in the same way as the sample config. Then I noticed there were a lot of plugins that already included this commented description in the sample config that they had in the README.md. So it seemed like added complexity in the tool and documentation, when we could just have it pre-included in the sample config for all plugins. I assume if we ever want to make use of the description method in the future for another reason, we could easily add it back by just parsing out the description comment from the sample configuration.

Reply:
This sounds fine by me! Thanks for the background